api/proto/attest/attest.proto

```diff
@@ -0,0 +1,58 @@
+// Copyright(c) 2017-2018 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+option go_package  = "github.com/lf-edge/eve/api/go/certs";
+option java_package = "com.zededa.cloud.uservice.proto";
+
+// This is the response payload for POST /api/v1/edgeDevice/attestNonce
+// or /api/v2/edgeDevice/attestNonce
+// The message is assumed to be protected by a TLS session bound to the
+// device certificate for v1.
+// The message is assumed to be protected by signing envelope for v2.
+
+message ZAttestNonceMsg {
```

Comment:
If the POST is empty it makes sense to say that in the comment.
And since this is the NonceResponse you can name it as such.

No need to have Msg at the end of the names IMHO.

Reply:
Dropped 'Msg' suffix